src/test/resources/regressions/features/trusted/trusted-functions.gobra

```diff
@@ -25,8 +25,8 @@ func positiveSumWithoutTrusted(x int, y int) (ret int) {
 
 trusted
 func functionWithNonSupportedFeature() int{
-	some non supported go feature
-	return 0
+	var i int = "not an integer"
```

Comment:
please add a comment that the body is parsed but not type-checked due to the `trusted` annotation (as opposed to not even parsed with the old parser)

Reply:
yes